src/main/java/appeng/tile/storage/TileChest.java

```diff
@@ -233,54 +226,28 @@ public int getCellCount()
 				{
 					double power = 1.0;
 
-					final ICellInventoryHandler<IAEItemStack> itemCell = cellHandler.getCellInventory( is, this,
-							AEApi.instance().storage().getStorageChannel( IItemStorageChannel.class ) );
-					final ICellInventoryHandler<IAEFluidStack> fluidCell = cellHandler.getCellInventory( is, this,
-							AEApi.instance().storage().getStorageChannel( IFluidStorageChannel.class ) );
-
-					if( itemCell != null )
+					for( IStorageChannel channel : AEApi.instance().storage().storageChannels() )
```

Comment:
![CRITICAL](https://sonarsource.github.io/sonar-github/severity-critical.png 'Severity: CRITICAL') Refactor this code to not nest more than 3 if/for/while/switch/try statements. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://sonar.orod.org/coding_rules#rule_key=squid%3AS134)